bloom-instance/service/backend/iam.tf

```diff
@@ -0,0 +1,47 @@
+
+resource "aws_iam_role" "service" {
+  name = "${local.resource_namespace}-ecs"
+
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      # Permit ECS to assume this role
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Principal = {
+          Service = "ecs-tasks.amazonaws.com"
+        }
+      }
+    ]
+  })
+}
+
+# resource "aws_iam_policy" "service_s3_access" {
```

Comment:
are these commented out sections for notes or just artifacts of development?

Reply:
These are placeholder comments.  At the time of the commit it didn't seem like backend would need to access S3 buckets, so I removed the policies that would allow that.  I commented rather than deleted so they could be easily added back in if that changed as Ines worked on the file upload code.